streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1243,6 +1245,50 @@ public void shouldRetryInitializationWhenLockExceptionInStateUpdater() {
         verify(stateUpdater).add(task01);
     }
 
+    @Test
+    public void shouldRetryInitializationWhenCanNotInitializeTask() {
```

Comment:
The test is very good!
I just have some formatting comments.

Reply:
Could you please rename the test to `shouldRetryInitializationWithBackoffWhenInitializationFails`?